### DISCUSSION TITLE
No new versions in Google Play Store

### DISCUSSION THREAD

**Discussion by swever826** [Bartuzen/qBitController#65] [Q&A] [answered]
First, thank you for this amazing app!

I've seen that the new versions after 0.6.0 are not being published in the Google Play Store. Is this a temporary issue or should I just download the app from the other places you have it available?

**Comment by Bartuzen** [comment, marked as answer]
Google has been rejecting my updates since 0.7.0 by saying that there is a problem with privacy policy, but they aren't telling me what exactly is wrong, so I couldn't fix the issue. But now, I believe I found the problem and sent an update to be reviewed. Hopefully this time, the update will be acccepted.

**Reply by swever826** [reply]
Google being google haha. Thank you for your response! I hope they let you publish the latest version soon. I'll try F-droid in the meantime.

**Reply by Bartuzen** [reply]
They finally approved the update, version 0.7.3 is now live on Google Play Store.
